python/nammu/controller/MenuController.py

```diff
@@ -34,6 +34,18 @@ def __init__(self, mainController):
         # Create view with a reference to its controller to handle events
         self.view = MenuView(self)
 
+    def enable_split_options(self, horizontal=True,
+                             vertical=True, arabic=True):
+        """
+        Set visibility of split menu items.
```

Comment:
Maybe replace with "Show split menu items as enabled or disabled" or something like that so it's more clear.